src/updaters/cargo-toml.ts

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {checkpoint, CheckpointType} from '../util/checkpoint';
+import {Update, UpdateOptions, VersionsMap} from './update';
+import {GitHubFileContents} from '../github';
+import * as TOML from '@iarna/toml';
```

Comment:
Hey, I used to work with @iarna 😄 I'm glad this is the TOML parser you landed on.

Reply:
this is the first I'm hearing of a PR adding a dependency that sparks joy! 😄 

(that toml package is also used in renovate, which I'm also drafting a PR for this week, so at least if it breaks, it'll break for everyone the same way!)